src/js/Properties.js

```diff
@@ -494,31 +494,16 @@ axs.properties.getTextFromHostLanguageAttributes = function(element,
                                                             existingComputedname,
                                                             recursive) {
     var computedName = existingComputedname;
-    if (axs.browserUtils.matchSelector(element, 'img')) {
-        if (element.hasAttribute('alt')) {
-            var altValue = {};
-            altValue.type = 'string';
-            altValue.valid = true;
-            altValue.text = element.getAttribute('alt');
-            if (computedName)
-                altValue.unused = true;
-            else
-                computedName = altValue.text;
-            textAlternatives['alt'] = altValue;
-        } else {
-            var altValue = {};
-            altValue.valid = false;
-            altValue.errorMessage = 'No alt value provided';
-            textAlternatives['alt'] = altValue;
-            var src = element.src;
-            if (typeof(src) == 'string') {
-                var parts = src.split('/');
-                var filename = parts.pop();
-                var filenameValue = { text: filename };
-                textAlternatives['filename'] = filenameValue;
-                computedName = filename;
-            }
-        }
+    if (axs.browserUtils.matchSelector(element, 'img') && element.hasAttribute('alt')) {
+        var altValue = {};
+        altValue.type = 'string';
+        altValue.valid = true;
+        altValue.text = element.getAttribute('alt');
+        if (computedName)
+            altValue.unused = true;
+        else
+            computedName = altValue.text;
+        textAlternatives['alt'] = altValue;
```

Comment:
Are we ok mixing dot notation with bracket notation for assignment? We're using both in this block of code.